common/app/common/configuration.scala

```diff
@@ -312,6 +312,9 @@ class GuardianConfiguration extends Logging {
     lazy val stripePublicToken =  configuration.getStringProperty("id.membership.stripePublicToken").getOrElse("")
     lazy val accountDeletionApiKey = configuration.getStringProperty("id.accountDeletion.apiKey").getOrElse("")
     lazy val accountDeletionApiRoot = configuration.getStringProperty("id.accountDeletion.apiRoot").getOrElse("")
+    lazy val optinCampaignLandingUrl = configuration.getStringProperty("id.opt-in-campaign-landing-url.url").getOrElse("https://gu.com/staywithus")
+    lazy val optinCampaignJourneyUrl = configuration.getStringProperty("id.opt-in-campaign-landing-url.url").getOrElse("https://profile.theguardian.com/consents/staywithus")
```

Comment:
would it be better to build this value (and the landing URL) from the other config values which I think already exist for `https://www.theguardian.com` and `https://profile.theguardian.com`